src/record.jl

```diff
@@ -96,11 +96,13 @@ end
 end
 
 @inline function setcell!(col::Array{String,1}, i, val::StrRange, str)
+    val.escapecount>0 && error("Not yet implemented")
     col[i] = alloc_string(str, val)
     PARSE_SUCCESS
 end
```

Comment:
I believe this whole method will now never be called anymore because `Vector{String}` should never use `StrRange` anymore. So we could probably just remove the whole method.

Reply:
Yes I think this is vestigial from before we started using StringArrays

Reply:
Alright, removed it.